xsd/src/schema_manager.js

```diff
@@ -237,7 +237,9 @@ SchemaManager.prototype.exportNamespaces = function() {
 	var self = this;
 	var namespacePrefixes = {};
 	var prefixUsed = {};
+	this.addNamespacePrefixes(namespacePrefixes, prefixUsed, {'xs': self.xsNS});
 	this.addNamespacePrefixes(namespacePrefixes, prefixUsed, self.originatingSchema.localNamespaces);
+	this.addNamespacePrefixes(namespacePrefixes, prefixUsed, {'': self.originatingSchema.targetNS});
```

Comment:
Please look at my other comment. This change is purely cosmetic for the final result, and has nothing to do with the bugfix.